schema/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE item (
+    id SERIAL PRIMARY KEY,
+    genre_id INT,
+    author_id INT,
+    label_id INT,
+    publish_date DATE,
+    archived BOOLEAN,
+    FOREIGN KEY (genre_id) REFERENCES genre(id),
+    FOREIGN KEY (author_id) REFERENCES author(id),
+    FOREIGN KEY (label_id) REFERENCES label(id)
+);
+
+CREATE TABLE game (  
+  id INT GENERATED ALWAYS AS IDENTITY,
+  multiplayer BOOLEAN,
+  last_played DATE,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE book (  
+  id INT GENERATED ALWAYS AS IDENTITY,
+  publisher VARCHAR(250),
+  cover_state VARCHAR(250),
+  PRIMARY KEY (id)
```

Comment:
A book is a child of the item and in the UML diagram book doesn't have an id attribute because is a child of the item so you have to create an id attribute but has to be referred to the item id. example:
 
CREATE TABLE book (  
  id INT,
  publisher VARCHAR(250),
  cover_state VARCHAR(250),
  FOREIGN KEY(id) REFERENCES item(id)
);

same case for the music_albums and game